### PR TITLE
demo commit

### PR DESCRIPTION
**Pull request by awhfahim**
added a new end point

### CODE REVIEW

**Comment by md-redwan-hossain** [comment]
Hello

File: WebApplication1/WebApplication1/Controllers/WeatherForecastController.cs [hunk 1]
```diff
@@ -3,7 +3,7 @@
 namespace WebApplication1.Controllers
 {
     [ApiController]
-    [Route("[controller]")]
+    [Route("Products")]
```
Comment (md-redwan-hossain) [inline WebApplication1/WebApplication1/Controllers/WeatherForecastController.cs]: problematic

File: WebApplication1/WebApplication1/Controllers/WeatherForecastController.cs [hunk 2]
```diff
@@ -29,5 +29,18 @@ public IEnumerable<WeatherForecast> Get()
             })
             .ToArray();
         }
+
```
Comment (md-redwan-hossain) [inline WebApplication1/WebApplication1/Controllers/WeatherForecastController.cs:32]: ok

File: WebApplication1/WebApplication1/Controllers/WeatherForecastController.cs (hunk 1 above)
Comment (awhfahim) [reply WebApplication1/WebApplication1/Controllers/WeatherForecastController.cs:6]: done

Comment (awhfahim) [reply WebApplication1/WebApplication1/Controllers/WeatherForecastController.cs]: done

**Pull request merged**